src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -148,38 +152,50 @@ class Filter extends Component<Props> {
   }
 
   renderWaysToBuy(filterState, mediator, counts) {
+    const ways = [
+      {
+        hasWorks: this.existy.hasBuyNowArtworks,
+        name: "Buy now",
+        state: "acquireable",
+      },
+      {
+        hasWorks: this.existy.hasMakeOfferArtworks,
+        name: "Make offer",
+        state: "offerable",
+      },
+      {
+        hasWorks: this.existy.hasAuctionArtworks,
+        name: "Bid",
+        state: "at_auction",
+      },
+      {
+        hasWorks: this.existy.hasForSaleArtworks,
+        name: "Inquire",
+        state: "inquireable_only",
+      },
+    ]
+
+    if (ENABLE_MAKE_OFFER !== "true") {
```

Comment:
I would avoid this check and just do a truthy check `if (ENABLE_MAKE_OFFER)`. That way, it can either be set to some value in Force env, or just be unset (`hokusai production unset ENABLE_MAKE_OFFER`), rather than needing to futz with a specific value. Just the presence should be sufficient.

This is also to avoid an annoyance between ENV vars (which all start out as strings), and the actual sharify values which already have a `JSON.parse` applied to them: https://github.com/artsy/force/blob/ab85589c1fd5c906b8561aadafacbc95250c90d2/src/desktop/config.coffee#L139

Also note that along with updating Reaction in Force with these changes, you'll need to add the `ENABLE_MAKE_OFFER` env var paperwork there (doesn't look like there is any such one yet, which makes me wonder how people are testing/QA'ing. Is there a lab feature or is it just assumed there will be no offerable works in production until its ready?).

Reply:
Gotcha - how does f5d30a5 look, is that what you were thinking?

Reply:
Oh and yes, I think you're right that there wasn't already a flag because production just doesn't have works in MO. So I guess this was the first change that required the flag!